### PR TITLE
 #comment add x-content-type-options cache-control and pragma headers

### PR DESCRIPTION
**Pull request by noodlemoodle**

### CODE REVIEW

**Review by jlhughes — APPROVED** [review]

File: src/foam/nanos/http/NanoRouter.java [hunk 1]
```diff
@@ -63,7 +63,12 @@ protected void service(final HttpServletRequest req, final HttpServletResponse r
     PM       pm         = new PM(this.getClass(), serviceKey);
 
     resp.setContentType("text/html");
-
+    resp.setHeader("X-Content-Type-Options", "nosniff");
```
Comment (kgrgreer) [inline src/foam/nanos/http/NanoRouter.java:66]: Please add one line comment for each of these to explain its purpose.

File: src/foam/nanos/http/NanoRouter.java [hunk 2]
```diff
@@ -63,7 +63,12 @@ protected void service(final HttpServletRequest req, final HttpServletResponse r
...
+    resp.setHeader("Cache-Control", "no-cache, no-store, must-revalidate");
```
Comment (kgrgreer) [inline src/foam/nanos/http/NanoRouter.java:67]: Is this line correct? Don't we want some level of caching to improve performance? Currently we're downloading some small images hundreds of times for the same client session.

Comment (noodlemoodle) [reply src/foam/nanos/http/NanoRouter.java:67]: it looks like the headers are only being applied to responses to post requests

File: src/foam/nanos/http/NanoRouter.java (hunk 1 above)
Comment (noodlemoodle) [reply src/foam/nanos/http/NanoRouter.java:66]: please review comments: https://github.com/foam-framework/foam2/pull/2390

**Pull request merged**